iroh-base/src/lib.rs

```diff
@@ -1,23 +1,45 @@
 //! Base types and utilities for Iroh
 #![cfg_attr(iroh_docsrs, feature(doc_cfg))]
 
+// TODO: remove
 #[cfg(feature = "base32")]
 #[cfg_attr(iroh_docsrs, doc(cfg(feature = "base32")))]
 pub mod base32;
+
+// TODO: move to own crate
+#[cfg(feature = "base32")]
+#[cfg_attr(iroh_docsrs, doc(cfg(feature = "base32")))]
+pub mod ticket;
+
 #[cfg(feature = "hash")]
-#[cfg_attr(iroh_docsrs, doc(cfg(feature = "hash")))]
-pub mod hash;
+mod hash;
 #[cfg(feature = "key")]
-#[cfg_attr(iroh_docsrs, doc(cfg(feature = "key")))]
-pub mod key;
+mod key;
 #[cfg(feature = "key")]
-#[cfg_attr(iroh_docsrs, doc(cfg(feature = "key")))]
-pub mod node_addr;
+mod node_addr;
 #[cfg(feature = "relay")]
-#[cfg_attr(iroh_docsrs, doc(cfg(feature = "relay")))]
-pub mod relay_map;
+mod relay_map;
 #[cfg(any(feature = "relay", feature = "key"))]
 mod relay_url;
-#[cfg(feature = "base32")]
-#[cfg_attr(iroh_docsrs, doc(cfg(feature = "base32")))]
-pub mod ticket;
+
+#[cfg(feature = "relay")]
+#[cfg_attr(iroh_docsrs, doc(cfg(feature = "relay")))]
+pub use self::relay_map::{
+    QuicConfig as RelayQuicConfig, RelayMap, RelayNode, DEFAULT_RELAY_QUIC_PORT, DEFAULT_STUN_PORT,
+};
+#[cfg(any(feature = "relay", feature = "key"))]
+#[cfg_attr(iroh_docsrs, doc(cfg(any(feature = "relay", feature = "key"))))]
```

Comment:
You should try this out, but I think if you reexport something from a module that has the `#[cfg_attr(iroh_docsrs, doc(cfg(any(feature = "relay", feature = "key"))))]` thing, all reexports will also have the proper tags in the docs.

So you could simplify this by just having this statement on the (non-pub) module.

Reply:
Ah, never mind. This is just iroh-base...

Reply:
Why do we need a const for the size of an ed key in the first place?

Reply:
> Why do we need a const for the size of an ed key in the first place?

we can drop that probably, need to check

Reply:
> Why do we need a const for the size of an ed key in the first place?

it is used in a few places, sorry

Reply:
Just write 32 or have a const per crate. It is not going to change.